src/hyp3_sdk/hyp3.py

```diff
@@ -264,8 +264,8 @@ def submit_rtc_job(self,
             resolution: Desired output pixel spacing in meters
             scale: Scale of output image; power, decibel or amplitude
             speckle_filter: Apply an Enhanced Lee speckle filter
-            dem_name: Name of the DEM to use for processing.  `copernicus` will use the Copernicus GLO-30 Public DEM,
-                while `legacy` will use the DEM with the best coverage from ASF's legacy SRTM/NED datasets.
+            dem_name: Name of the DEM to use for processing.  `copernicus` is the only option, and it will use
+            the Copernicus GLO-30 Public DEM.
```

Comment:
```suggestion
            dem_name: Name of the DEM to use for processing. `copernicus` is the only option, and it will use
            the Copernicus GLO-30 Public DEM.
```